recipes-bsp/bootfiles/rpi-cmdline.bb

```diff
@@ -34,7 +34,13 @@ CMDLINE_DEBUG ?= ""
 # CMDLINE_RNDIS = "modules-load=dwc2,g_ether g_ether.host_addr=<some MAC 
 # address> g_ether.dev_addr=<some MAC address>"
 # if the MAC addresses are omitted, random values will be used
-CMDLINE_RNDIS ?= ""
+CMDLINE_RNDIS ?= '${@oe.utils.conditional("ENABLE_RNDIS", "1", "modules-load=dwc2,g_ether", "", d)}'
```

Comment:
Isn't this dependent on the kernel configuration? module/built-in that is.

Reply:
Yes it does, this requires that the OTG setting is also enabled. I will elaborate the dependencies to clarify this.

Reply:
Done